lib/element-handlers/addPcbBoard/index.ts

```diff
@@ -39,9 +39,15 @@ export const addPcbBoard = (board: PcbBoard, ctx: ConvertContext) => {
 
   const { verts, prims } = polygonToShapePathData(polygon)
 
+  // Determine which cut setting to use based on the preset field
+  const cutSetting =
+    (board as any).preset === "soldermask_cutout"
```

Comment:
preset does not exist

Reply:
😁 appreciated- you're not a big offender here but some others are so i'm slowly getting more anal